app/views/users/_photo.html.erb

```diff
@@ -35,6 +35,15 @@
 </div>
 <p class="profile-uploading" style="display:none;padding-top:4px;color:#aaa;"><%= t('users._photo.uploading') %>...</p>
 
+<div style="font-size: 20px; text-align: center; margin-top: -20px;">
+  <% if @user.photo %>
```

Comment:
I think, here check should be @user.photo_file_name as currently, option is visible to every user